### DISCUSSION TITLE
No Menu in WSL (Ubuntu)

### DISCUSSION THREAD

**Discussion by aagha** [saulpw/visidata#2347] [Q&A]
Going through the tutorial, I tried to use ctrl+h to go to the menu, but it instead takes me to the quick reference guide:

![image](https://github.com/saulpw/visidata/assets/5522103/72df6b34-9b18-45b0-aaf3-af3e5ba91974)

Going through the commands (Shift+O), there is no `disp_menu` option:

![image](https://github.com/saulpw/visidata/assets/5522103/30e7d7d6-ff31-41f4-9fea-61bb43df7eb5)

How do I access the menu?

**Comment by aagha** [comment]
Looking further, I found this post:

https://github.com/saulpw/visidata/issues/183#issuecomment-416369185

In my case, it seems both backspace and ctrl-h are going to the Quick Reference Guide, so it seems that backspace and ctrl-h are both getting mapped together.

I can try creating the .visidatarc as pur a subsequent suggestion.  I'm also wondering if it's possible to map the keys in Terminal:

![image](https://github.com/saulpw/visidata/assets/5522103/c9c281da-d4a2-46a6-9887-dbb139141c78)

**Reply by aagha** [reply]
I'v added the followign to .visidatarc

`bindkey('^H', 'help-commands')` -- As a short-cut, but this doesn't get me the menu.

**Reply by anjakefala** [reply]
Your version of VIsiData is too old! The menu was added in v2.6: https://www.visidata.org/blog/2021/v2.6/

**Reply by aagha** [reply]
Oh my goodness! I had 1.5.2!  I have no idea how/why.

I upgraded to 3.0.2. Thanks!
